decidim-admin/app/views/decidim/admin/participatory_process_user_roles/_table.html.erb

```diff
@@ -0,0 +1,34 @@
+<section id="process_admins">
+  <h4><%= t(".process_admins_title", scope: "decidim.admin") %></h4>
+
+  <%= form_for :participatory_process_user_role, url: participatory_process_user_roles_path(@participatory_process) do |f| %>
+    <table class="stack">
+      <thead>
+        <tr>
+          <th><%= t("models.participatory_process_user_role.fields.email", scope: "decidim.admin") %></th>
```

Comment:
This implies changing all locales. Also, this will try to look for locales in this route:

```
activerecord.attributes.<model>.<field>
```

Do we really want the fields translations outside of the `decidim` scope? What if we want/need a different translation for the same attribute in different engines?